### ISSUE TITLE
MacOS screencapture not working properly?

### ISSUE THREAD

**Issue by shirshak55** [nashaofu/xcap#123]
It only shows background window.
Brave is running, but I don't why it isn't capturing it.
<img width="1101" alt="image" src="https://github.com/nashaofu/xcap/assets/8097377/2d7e07c1-151d-43db-9087-13ab086299db">

**Comment by nashaofu** [comment]
Do you have screenshot permission for your application? Without screenshot permission, you can only take screenshots of the desktop

**Comment by shirshak55** [comment]
Yes. I have given permission. However, it keeps asking permission

**Comment by ambethia** [comment]
Same issue here on macos. I have also granted permission to the application.

**Comment by codesoda** [comment]
@shirshak55 Did the solution mentioned above work for you?

**Comment by shirshak55** [comment]
@codesoda not i am not running from visual studio code. I am running inside desktop app via tauri. If I run it from terminal it works, but when I run it as desktop app it doesn't.

**Comment by nashaofu** [comment]
https://github.com/nashaofu/tauri-xcap-demo
I am able to take screenshots normally and command an authorization prompt to pop up once @shirshak55 @codesoda

**Comment by nashaofu** [comment]
https://github.com/nashaofu/xcap/assets/19303058/cdaa788e-9587-4836-a913-34c75ef26b4d

**Comment by shirshak55** [comment]
@nashaofu 

Today I ran same old rust code and I am not facing the issue. I think macos updates fixed something or something happened without changing the code.

**Comment by shirshak55** [comment]
@nashaofu 

Ok, I think it is because of github action. If you built using github action, can u check if it works flawlessly?

**Comment by nashaofu** [comment]
https://github.com/nashaofu/tauri-xcap-demo/releases/tag/app-v0.0.0. Can you give it a try? I'm working normally here
